PR: https://github.com/odoo/odoo/pull/178200

From: f10bd8ddcf99c9247dca417fefbc260454d74ea1
From: Gorash
Date: 2024-10-15 09:33:02

Breaking data model changes scores: del:4 + add:0, change matches:
-    _inherit = 'event.type.mail'
-    _inherit = 'event.mail'
-    _inherit = 'event.mail.registration'
-    _inherit = 'sms.template'

Total Changes: 10

[IMP] *: adapt model class names to correspond to model names (apply script)

Part-of: odoo/odoo#178200
Related: odoo/enterprise#69762
Related: odoo/documentation#11115
Related: odoo/upgrade#6577
Related: odoo/design-themes#988
Related: odoo/upgrade-util#143

================================= pseudo patch: =================================

--- a/addons/event_sms/models/event_mail.py
+++ b/addons/event_sms/models/event_mail.py
@@ -5,7 +5,7 @@ from odoo import fields, models
 
 
 class EventTypeMail(models.Model):
-    _inherit = 'event.type.mail'
+    _inherit = ['event.type.mail']
 
     notification_type = fields.Selection(selection_add=[('sms', 'SMS')])
     template_ref = fields.Reference(ondelete={'sms.template': 'cascade'}, selection_add=[('sms.template', 'SMS')])
@@ -16,8 +16,8 @@ class EventTypeMail(models.Model):
         sms_schedulers.notification_type = 'sms'
 
 
-class EventMailScheduler(models.Model):
-    _inherit = 'event.mail'
+class EventMail(models.Model):
+    _inherit = ['event.mail']
 
     notification_type = fields.Selection(selection_add=[('sms', 'SMS')])
     template_ref = fields.Reference(ondelete={'sms.template': 'cascade'}, selection_add=[('sms.template', 'SMS')])

--- a/addons/event_sms/models/event_mail_registration.py
+++ b/addons/event_sms/models/event_mail_registration.py
@@ -2,7 +2,7 @@ from odoo import fields, models
 
 
 class EventMailRegistration(models.Model):
-    _inherit = 'event.mail.registration'
+    _inherit = ['event.mail.registration']
 
     def _execute_on_registrations(self):
         todo = self.filtered(

--- a/addons/event_sms/models/sms_template.py
+++ b/addons/event_sms/models/sms_template.py
@@ -6,7 +6,7 @@ from odoo.osv import expression
 
 
 class SmsTemplate(models.Model):
-    _inherit = 'sms.template'
+    _inherit = ['sms.template']
 
     @api.model
     def _search_display_name(self, operator, value):
